cot_config_example.json

```diff
@@ -0,0 +1,21 @@
+{
+    "git_key_repo_url": "https://github.com/mozilla-releng/cot-gpg-keys.git",
+    "git_commit_signing_pubkey_dir": "...",
```

Comment:
Sure, though `/tmp/gpg/` will get blown away because it's the `base_gpg_home_dir` in `config_example.json`.